client/src/pages/Plan/CoursePill.js

```diff
@@ -15,10 +15,17 @@ const CoursePill = ({ courseName }) => {
     const opacity = isDragging ? 0.4 : 1;
 
     return (
-        <Box flex="0 0 100%" borderRadius="10px" borderRight="solid white" maxWidth="33%" marginTop="10px" style={{ opacity }}>
-            <Flex p={2} cursor="pointer" width="90%" backgroundColor={c.midnightBlue} borderRadius="8px" ref={drag}>
+        <Box flex="0 0 100%" borderRadius="10px" borderRight="solid white" maxWidth="50%" marginTop="10px" style={{ opacity }}>
+            <Flex
+                p={2}
+                cursor="pointer"
+                width="90%"
+                backgroundColor={isPlaceholder ? c.lightBlue : c.midnightBlue}
```

Comment:
Can we change the colour to `c.uoaBlue`?

```suggestion
                backgroundColor={isPlaceholder ? c.uoaBlue : c.midnightBlue}
```